manager/testdata/new-blueprints/copy-read-blueprint.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: Blueprint
+metadata:
+  name: copy-and-read
+  labels:
+spec:
+  cluster: theshire
+  appSelector: "app:demoapp"
+  appCluster: theshire
+  flow:
+  - name: copy-step
+    assetID: 'm4d-notebook-sample/paysim-csv'
+    templateSpec:
+      chart: ghcr.io/mesh-for-data/m4d-implicit-copy-batch:0.1.0
+    moduleLabel: "mod: demoapp-copy-step"
+    frequency: "0 1 * * *"
+    source:
+      vault:
+        address: http://vault.m4d-system:8200
+        authPath: /v1/auth/kubernetes/login
+        role: module
+        secretPath: /v1/kubernetes-secrets/paysim-csv?namespace=m4d-notebook-sample
+      format: parquet
+      connection:
+        s3:
+          endpoint: localhost:8001
+          bucket: srcbucket
+          object: data.parq
+    destination:
+      vault:
+        address: http://vault.m4d-system:8200
+        authPath: /v1/auth/kubernetes/login
+        role: module
+        secretPath: /v1/kubernetes-secrets/paysim-csv?namespace=m4d-notebook-sample
+      format: parquet
+      connection:
+        s3:
+          endpoint: localhost:8001
+          bucket: destbucket
+          object: data.parq
+    transformations:
+    - action: redact
+      column: blood_group
+  - name: read-step
+    templateSpec:
+      chart: ghcr.io/mesh-for-data/m4d-arrow-flight:0.1.0
+    moduleSelector: "mod: demoapp-read-step"
+    assetID: 'm4d-notebook-sample/paysim-csv'
+    source:
+      step: copy-step # the name of step for debugging purposes
+      selector: "mod: demoapp-copy-step" # Selector of the dependency
```

Comment:
Can there be multiple dependencies?

Reply:
When we discussed it before we said that we do not merge or combine two datasets for governance enforcement purposes.